std/algorithm/iteration.d

```diff
@@ -62,6 +66,7 @@ module std.algorithm.iteration;
 
 // FIXME
 import std.functional; // : unaryFun, binaryFun;
+import std.range;  // : SortedRange;
 import std.range.primitives;
```

Comment:
if you import all of std.range why do you still import std.range.primitives ?

only import what you really need.